### DISCUSSION TITLE
How do I enable email notifications for discussions?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#1000] [Ask a question] [answered]
Is there an easy way for us to be notified when a new question is posted here? (by email, like in a newsletter).

**Comment by annaesvensson** [comment, marked as answer]
Go to the main page of [datenstrom/community](https://github.com/datenstrom/community), log in with your user account, in the top-right corner click **Watch**.

Now you will receive notifications for all discussions. You can enable notifications for any GitHub repository you are interested in. You can view notifications in your email client and on the GitHub website. It's also possible to view notifications in the GitHub mobile app, have a look at the GitHub documentation, there are a quite a few configuration options.

Edit: Updated link

**Reply by annaesvensson** [reply]
Did this answer your question, @yesilovebooks?

**Reply by ghost** [reply]
yes, thank you!

**Reply by ghost** [reply]
> Go to the main page of [datenstrom/yellow](https://github.com/datenstrom/yellow), in the top-right corner click **Watch**.
> 
> Now you will receive notifications for all discussions. You can enable notifications for any GitHub repository you are interested in. You can view notifications in your email client and on the GitHub website. It's also possible to view notifications in the GitHub mobile app, have a look at the GitHub documentation, there are a quite a few configuration options: https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications

Hello, @annaesvensson !

I've followed the steps, checked the "Discussions" box and now I will be notified for all new posts here.

Thank you!
